### PR TITLE
v0.3 release information added to the timeline in Home screen

### PR DESCRIPTION
**Pull request by nehraankit**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by anvithks** [comment]
/hold
@nehraankit please rebase your branch and raise the PR. There are older commits in this one.

File: content/en/home/timeline.md [hunk 1]
```diff
@@ -18,6 +18,23 @@ date_format = "January 2006"
...
+  location = "May, 2019"
+  date_start = ""
+  date_end = ""
+  description = """
+**Features added**  
+
+- Device Management using CRDs feature mainly provides APIs for managing devices from cloud and synchronize the device updates between cloud and edge <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/device-crd.md' target="_blank">&hellip; Read more</a>
+- Bluetooth Mapper application support which is used to connect and control devices that use bluetooth as a communication protocol.
+- Performance Test Framework to measure the performance against the Service Level Objectives like Latency, Throughput, Scalability, CPU Usage, Memory Usage etc <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/perf.md' target="_blank">&hellip; Read more</a>
+- KubeEdge Installer support for basic commands like "kubeedge init", "kubeedge join" and "kubeedge reset" to bootstrap and teardown both KubeEdge cloud and edge components.
```
Comment (rohitsardesai83) [inline content/en/home/timeline.md]: let's add a link for installer md file as well : https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/keadm-scope.md

**Pull request closed without merging**